src/shared/components/layouts/styles/side-bar-layout.scss

```diff
@@ -42,7 +42,6 @@
 
   &__content {
     flex: 1;
-    position: sticky;
     top: $header-height;
     right: 0;
```

Comment:
I think none of this is needed now.
flex: 1 only if we want the table to stretch when not a lot of columns but a really wide screen, which is questionable

(misplaced the previous comment which I just deleted)

Reply:
I think you're right...so this was an attempt at fixing the overflowing table?

Reply:
I don't really know anymore, maybe